Doc/license.rst

```diff
@@ -383,33 +383,6 @@ Project, https://www.wide.ad.jp/. ::
    SUCH DAMAGE.
 
 
-Asynchronous socket services
-----------------------------
-
-The :mod:`asynchat` and :mod:`asyncore` modules contain the following notice::
```

Comment:
I would prefer to keep the mention since ``Lib\test`` is part of the stdlib on some operating systems, so the license matters. I just asked to mention that asynchat and asyncore are now sub-modules of the ``test.support`` package.